### PR TITLE
Feature/default timeout

### PR DESCRIPTION
**Pull request by pengin7384**
Add provider send's default timeout time

### CODE REVIEW

File: src/provider.ts [hunk 1]
```diff
@@ -49,4 +51,12 @@ export default class Provider {
       return null;
     }
   }
+
+  /**
+   * Sets the httpClient's default timeout time
+   * @param {number} time
```
Comment (liayoo) [inline src/provider.ts:57]: Could you also add to the comment if this is in seconds or milliseconds? And maybe rename the function to 'setDefaultTimeoutMs' if it's in milliseconds?

Comment (pengin7384) [reply src/provider.ts]: Looks better! Done

**Review by liayoo — APPROVED** [review]
LGTM. Thanks!

**Comment by pengin7384** [comment]
Thanks for the review!

**Pull request merged**